Mathlib/CategoryTheory/Category/GaloisConnection.lean

```diff
@@ -28,8 +28,10 @@ def GaloisConnection.adjunction {l : X → Y} {u : Y → X} (gc : GaloisConnecti
     gc.monotone_l.functor ⊣ gc.monotone_u.functor :=
   CategoryTheory.Adjunction.mkOfHomEquiv
     { homEquiv := fun X Y =>
-        ⟨fun f => CategoryTheory.homOfLE (gc.le_u f.le),
-         fun f => CategoryTheory.homOfLE (gc.l_le f.le), _, _⟩ }
+        { toFun := fun f => CategoryTheory.homOfLE (gc.le_u f.le)
```

Comment:
This is a small readability regression to me, but only a small one -> I won't object to landing this.